pkg/controller/keyvault/consts.go

```diff
@@ -3,4 +3,14 @@ package keyvault
 const (
 	tlsCertKvUriAnnotation   = "kubernetes.azure.com/tls-cert-keyvault-uri"
 	tlsCertManagedAnnotation = "kubernetes.azure.com/tls-cert-keyvault-managed"
+
+	istioGatewayClassName = "istio"
```

Comment:
is this the gateway class name that istio always uses is this what the istio just happens to use?

Reply:
It's the gateway class used by istio - see here: https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#configuring-a-gateway